includes/wpmenucart-woocommerce.php

```diff
@@ -56,7 +73,7 @@ public function shop_url() {
 		}
 
 		public function get_cart_contents_count() {
-			if ( function_exists( 'WC' ) ) {
+			if ( function_exists( 'WC' ) && ! empty( WC()->cart ) ) {
```

Comment:
this conditional is used to differentiate between legacy versions of WC (which didn't have the `WC()` function). By adding the `! empty()` call in this same condition, you're forcing callback to the `$woocommerce` global, effectively making the same call, this is ineffective.